task.go

```diff
@@ -330,7 +330,7 @@ func (config TaskConfig) validateInputContainsNames() []string {
 type TaskRunConfig struct {
 	Path string   `json:"path" yaml:"path"`
 	Args []string `json:"args,omitempty" yaml:"args"`
-	Dir  string   `json:"dir",omitempty" yaml:"dir"`
```

Comment:
When i ran the unit test locally it did not break things. It probably should be noted in a release cause it could be broken....